From 18166b655667271c7749537a7b9af8f82125628e Mon Sep 17 00:00:00 2001
From: Zhen Chen <chenzhen@rock-chips.com>
Date: Tue, 7 Aug 2018 09:34:12 +0800
Subject: [PATCH] Mali: midgard: set "FMODE_UNSIGNED_OFFSET" bit in file flags
 of mali device file

According to commit "mmap: introduce sane default mmap limits"

Change-Id: I96bc78053c06fdeaef178662dda36770da867436
Signed-off-by: Zhen Chen <chenzhen@rock-chips.com>
---
 drivers/gpu/arm/midgard/mali_kbase_core_linux.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/arm/midgard/mali_kbase_core_linux.c b/drivers/gpu/arm/midgard/mali_kbase_core_linux.c
index 3a6e5aae0bce..24bafe2bf32c 100644
--- a/drivers/gpu/arm/midgard/mali_kbase_core_linux.c
+++ b/drivers/gpu/arm/midgard/mali_kbase_core_linux.c
@@ -1154,6 +1154,7 @@ static int kbase_open(struct inode *inode, struct file *filp)
 	}
 
 	init_waitqueue_head(&kctx->event_queue);
+	filp->f_mode |= FMODE_UNSIGNED_OFFSET;
 	filp->private_data = kctx;
 	kctx->filp = filp;
 
-- 
2.35.3

